bin/node/runtime/src/lib.rs

```diff
@@ -1012,7 +1012,7 @@ construct_runtime!(
 	{
 		System: frame_system::{Module, Call, Config, Storage, Event<T>},
 		Utility: pallet_utility::{Module, Call, Event},
-		Babe: pallet_babe::{Module, Call, Storage, Config, Inherent, ValidateUnsigned},
+		Babe: pallet_babe::{Module, Call, Storage, Config, ValidateUnsigned},
```

Comment:
You did not added `Babe` to the `OnTimestampSet` associated types :D

Reply:
I think it should already be there.

Reply:
yes somehow Babe was in `OnTimestampSet` for all runtimes (same in polkadot companion)

Reply:
Ahh :D